types/src/index.d.ts

```diff
@@ -175,6 +176,25 @@ export type PROVIDER_ID =
   | "ForkedGithubRepoProvider"
   | "StarredGithubRepoProvider"
   | "FiveOrMoreGithubRepos"
```

Comment:
```suggestion
  | "FiveOrMoreGithubRepos"
  | "Gitcoin"
```

Reply:
We're not issuing a Stamp for Gitcoin on its own (it would be the same as the github stamp), so "Gitcoin" shouldn't need to be a `PROVIDER_ID` I don't think?

Reply:
> We're not issuing a Stamp for Gitcoin on its own (it would be the same as the github stamp), so "Gitcoin" shouldn't need to be a `PROVIDER_ID` I don't think?

Ahhh yes, that makes sense!